package Practice.ExceptionHandling.Durga.Need_of_throw_keyword;

public class Case1
{
    public static void main(String[] args) {

        System.out.println("Statement-1");
        throw new ArithmeticException("I have created Exception-Object to terminate Program abnormally!!!");
    }
}

